ci/test_wheel.sh

```diff
@@ -9,23 +9,26 @@ mkdir -p ./dist
 RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
 RAPIDS_PY_WHEEL_NAME="pylibwholegraph_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./dist
 
-# echo to expand wildcard before adding `[extra]` requires for pip
-python -m pip install $(echo ./dist/pylibwholegraph*.whl)
-
+# determine PyTorch source
 PKG_CUDA_VER="$(echo ${CUDA_VERSION} | cut -d '.' -f1,2 | tr -d '.')"
 PKG_CUDA_VER_MAJOR=${PKG_CUDA_VER:0:2}
 if [[ "${PKG_CUDA_VER_MAJOR}" == "12" ]]; then
   INDEX_URL="https://download.pytorch.org/whl/cu121"
 else
   INDEX_URL="https://download.pytorch.org/whl/cu${PKG_CUDA_VER}"
 fi
+
+# echo to expand wildcard before adding `[extra]` requires for pip
+python -m pip install \
+  -v \
+  --extra-index-url "${INDEX_URL}" \
+  "$(echo ./dist/pylibwholegraph_${RAPIDS_PY_CUDA_SUFFIX}*.whl)[test]" \
+  'torch>=2.0,<2.4.0a0'
```

Comment:
I've asked about that before and @alexbarghi-nv mentioned to me that `torch` is intentionally kept out of some of the lists in `dependencies.yaml`, I think to avoid DLFW having to patch it out? I'm struggling to find the link to that conversation right now.

But anyway... keeping it separate is something that was done intentionally, and this PR doesn't have to be the one to fix that.